### PR TITLE
Separate commands and output

### LINKED ISSUE #209: Ep4: Commands and output mixed into same code box
Near the bottom we have 

```
$ touch hello.txt && tar -cvf hello.tar hello.txt
$ cwl-runner tar.cwl tar-job.yml
[job tar.cwl] /tmp/tmpqOeawQ$ tar \
    --extract --file \
. . . 
```
It would be clearer to separate the code and the output into separate  code blocks.

### PR DESCRIPTION
**Pull request by kinow**
Closes #209

### CODE REVIEW

File: src/topics/outputs.md [hunk 1]
```diff
@@ -35,11 +35,16 @@ any subsequent elements are mandatory command line arguments
...
-Next, create a tar file for the example and invoke `cwl-runner` with the tool
-wrapper and the input object on the command line:
+Next, create a tar file for the example.
 
 ```{code-block} console
 $ touch hello.txt && tar -cvf hello.tar hello.txt
+hello.txt
+```
+
+And now invoke `cwl-runner` with the tool wrapper and the input object on the command line:
```
Comment (mr-c) [inline src/topics/outputs.md:45]: An addition to the style guide, use "tool description" not "tool wrapper"

Comment (kinow) [reply src/topics/outputs.md:45]: TIL, will fix it and update the style guide too. Thanks!

Comment (kinow) [reply src/topics/outputs.md]: Done!

Comment (mr-c) [reply src/topics/outputs.md]: descript**ion** not descript**or** :-)

**Review by mr-c — APPROVED** [review]
Thanks!

File: src/topics/outputs.md (hunk 1 above)
Comment (kinow) [reply src/topics/outputs.md]: Yeah, I went to do a quick self-edit of what I wrote, and realized it. Then I tried fixing that after rebasing, and wrote “Use tool description, not tool description”. Then fixed that and messed up somewhere else. Argh. Hopefully fixed now :crossed_fingers:

**Pull request merged**